Makefile

```diff
@@ -92,17 +104,26 @@ ut-containerized: vendor
 
 GET_CONTAINER_IP := docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}'
 K8S_VERSION=1.7.5
+
+ifeq ($(ARCH),amd64)
```

Comment:
Could this be simplified by using `$ARCHTAG`?  That might be nicer.

e.g.

```
ETCD_CONTAINER?=quay.io/coreos/etcd:v3.2.5$(ARCHTAG)
```

Reply:
Unfortunately both the version and name of the etcd container on quay.io is different between amd64 and ppc64le.  Ppc64le never had a v3.1.5.b So we still need a ifeq $(ARCH) to select the version.  If we switched to v3.2.5 for both we could eliminate the ifdef. However if in the future the version or location should change (or another arch is added) we are back to ifeq $(ARCH).

Reply:
There's no requirement that etcd be pinned to v3.1.5 for this target.  I think it's preferable to use the same version for all values of ARCH anyway.  I think for now let's put both of them to `v3.2.5` and if we need to re-introduce the ifdef down the road we can do that.